pkg/minikube/registry/drvs/kvm2/kvm2.go

```diff
@@ -113,16 +114,55 @@ func status() registry.State {
 		return registry.State{Error: err, Fix: "Install libvirt", Doc: docURL}
 	}
 
+	// check if the current user is a member of "libvirt*" group
+	lvMember := false
```

Comment:
May be add a function for this check? Just to make `status()` still fit in in one screen. something like `curUserLibvirt() (bool, error)`? Need to find a better name though :)

Reply:
agree with @ilya-zuyev , lets add a helper function that this get called, for better code readablity @prezha

Reply:
@ilya-zuyev , @medyagh thank you for your review: i agree and moved the check into a separate helper func